adapter/operations.go

```diff
@@ -14,6 +14,13 @@
 
 package adapter
 
+const (
+	OperationDescriptionKey  = "description"
```

Comment:
`description` is used in api/grpc/handlers.go:76, and should be replaced by this constant.

Reply:
done, thanks for noticing this